src/org/labkey/test/pages/core/admin/LimitActiveUserPage.java

```diff
@@ -0,0 +1,115 @@
+package org.labkey.test.pages.core.admin;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.components.html.Input;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+
+public class LimitActiveUserPage extends LabKeyPage<LimitActiveUserPage.ElementCache>
+{
+    public LimitActiveUserPage(WebDriver driver)
+    {
+        super(driver);
+        waitForPage();
+    }
+
+    public static LimitActiveUserPage beginAt(WebDriverWrapper wrapper)
+    {
+        wrapper.beginAt(WebTestHelper.buildURL("user", "limitActiveUsers"));
+        return new LimitActiveUserPage(wrapper.getDriver());
+    }
+
+    public LimitActiveUserPage userWarning(String value)
+    {
+        elementCache().userWarning.selectByVisibleText(value);
+        return this;
+    }
```

Comment:
I would change this parameter to a boolean because you are turning the property on or off, the string parameter doesn't really get you anything. Then set the text to select in the function.

```
public LimitActiveUserPage userWarning(boolean value)
{
    String setValue = boolean ? "Yes" : "No";
    elementCache().userWarning.selectByVisibleText(setValue);
    return this;
}
```

Reply:
Just curious as to why does it has to be boolean?

Reply:
It doesn't _have to_ be a boolean, I think it works better as a boolean. Heres why:

- Boolean makes it easier for someone who is not familiar with the feature to turn the feature on (true) or off (false).
- A string gives no indication of the possible values that can be sent in. Boolean removes ambiguity.
- A string allows for typos ('yes' vs 'Yes').
- A boolean insulates more from product changes. For example if the value changes from 'Yes'/'No' to 'On'/'Off' only the method on the page has to change not the test cases.

Reply:
Done..! Changed